python-avd/pyavd/_eos_designs/eos_designs_facts/uplinks.py

```diff
@@ -369,9 +369,6 @@ def _get_p2p_vrfs_uplink(
                     subinterface.ip_address = self.shared_utils.ip_addressing.p2p_vrfs_uplinks_ip(uplink_index, vrf.name)
                     subinterface.peer_ip_address = self.shared_utils.ip_addressing.p2p_vrfs_uplinks_peer_ip(uplink_index, vrf.name)
 
-                if self.shared_utils.node_config.uplink_structured_config is not None:
```

Comment:
it seems we don't have a replacement for this with the newer keys (having structured_configuration on the subinterfaces) need to recheck why we decided to not implement this

Reply:
checked the code and actually this was never used in `underlay/ethernet_interfaces.py` so all good